File: Python Code/Daily_Flash_Code_0/Week11/2Apr/63-DailyFlash_Solutions/2-Apr_SOlution_One/Java/prog5.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		float T;
		Scanner sc = new Scanner(System.in);
		T = sc.nextFloat();
		float l = sc.nextFloat();
		System.out.print("g: ");
		System.out.println((l * T) / (4 * Math.pow(3.14, 4)));
	}
}

File: Python Code/Daily_Flash_Code_0/Week11/2Apr/63-DailyFlash_Solutions/2-Apr_SOlution_One/C/prog5.c
#include<stdio.h>
#include<math.h>



int main() {
	float T, l;
	printf("T L: ");
	scanf("%f %f", &T, &l);
	printf("a = %f", (l * T) / (4 * pow(3.14, 4)));
}
